src/zyppinstall.cpp

```diff
@@ -101,11 +101,11 @@ bool ZyppInstall::commitChanges()
 		//				 ERR_COMMIT : ERR_ZYPP );
 	    throw "Incomplete Commit";
 	}
-	cout << "Commit Succeeded" << endl;
+	qDebug() << "Commit Succeeded";
     }
     catch ( const media::MediaException& e)
     {
-	string errDescription = "Problem retrieving the package file from the repository";
+	string errDescription = i18n( "Problem retrieving the package file from the repository" ).toStdString();
 	//OCIHelper::instance()->setErrorDescription( ERR_COMMIT, errDescription );
```

Comment:
Need a lot of thinking and time to be put into exception handling. I'll create a new PR once all the exception handling is done for OCIHelper.
Exception handling in this case is to catch the exception, process it, display appropriate message to the user, and exit both applications (OCI and OCIHelper) safely.